spinnman/processes/get_machine_process.py

```diff
@@ -164,10 +164,14 @@ def get_machine_details(self, boot_x, boot_y, width, height):
         def chip_xy(chip):
             return chip.x, chip.y
 
+        import time
+        t1 = time.time()
         chips = [
             self._make_chip(width, height, chip_info)
             for chip_info in sorted(self._chip_info.values(), key=chip_xy)
             if (chip_info.x, chip_info.y) not in self._ignore_chips]
+        t2 = time.time()
+        print("chips :", (t2 - t1))
```

Comment:
really? you want to print this for every end user???

Reply:
Sorry test code please nuke the three time lines